### ISSUE TITLE
[Code generator] Exceeds the limit (4300) for integer string conversion

### ISSUE THREAD

**Issue by NeoQuix** [fkie-cad/dewolf#216]
### What happened?

```python
Error in bin/services.exe in 0x14006b590
Traceback (most recent call last):
  File "/home/neoquix/Git-Repos/DeWolf/decompile.py", line 76, in <module>
    main(Decompiler)
  File "/home/neoquix/Git-Repos/DeWolf/decompiler/util/commandline.py", line 80, in main
    task = decompiler.decompile(function_name, options)
  File "/home/neoquix/Git-Repos/DeWolf/decompile.py", line 52, in decompile
    task.code = self._backend.generate([task])
  File "/home/neoquix/Git-Repos/DeWolf/decompiler/backend/codegenerator.py", line 28, in generate
    string_blocks.append(GlobalDeclarationGenerator.from_asts(task.syntax_tree for task in tasks if not task.failed))
  File "/home/neoquix/Git-Repos/DeWolf/decompiler/backend/variabledeclarations.py", line 95, in from_asts
    return "\n".join(generator.generate())
  File "/home/neoquix/Git-Repos/DeWolf/decompiler/backend/variabledeclarations.py", line 100, in generate
    yield f"extern {variable.type} {variable.name} = {self.get_initial_value(variable)};"
  File "/home/neoquix/Git-Repos/DeWolf/decompiler/backend/variabledeclarations.py", line 129, in get_initial_value
    return str(convert_bytes(variable.initial_value, variable.type))
ValueError: Exceeds the limit (4300) for integer string conversion; use sys.set_int_max_str_digits() to increase the limit
```

### How to reproduce?

Decompile [services](https://github.com/fkie-cad/dewolf/files/11169059/services.tar.gz) at 0x14006b590

### Affected Binary Ninja Version(s)

3.3.3996

**Comment by NeoQuix** [comment]
Fixed with #233

**Comment by NeoQuix** [comment]
Fixed by #233 merge
